src/PolygonDraw/Polygon.tsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import { Polygon as LeafletPolygon } from 'react-leaflet';
+
+import { Coordinate } from 'types';
+import { createLeafletLatLngFromCoordinate } from '../helpers';
+import { MAP } from '../constants';
+
+interface Props {
+    coordinates: Coordinate[];
+    isActive: boolean;
+    onClick: () => void;
+    onMouseEnter: () => void;
+    onMouseLeave: () => void;
+}
+
+export const Polygon: React.FC<Props> = ({ isActive, coordinates, onClick, onMouseEnter, onMouseLeave }) => (
+    <LeafletPolygon
+        positions={coordinates.map(createLeafletLatLngFromCoordinate)}
+        dashArray={isActive ? '' : '4 12'}
```

Comment:
Question: what does '4 12' refer to? I would prefer to have it in const that explain what's is actually and why it's '4 12'

Reply:
Added const and comment for that in next commit. 
```
Refers to https://leafletjs.com/reference-1.6.0.html#path-dasharray and https://developer.mozilla.org/de/docs/Web/SVG/Attribute/stroke-dasharray
4 describes the dash size, 12 the gap size
```